lib/formatters/unix.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @fileoverview unix-style formatter.
+ * @author Nicholas C. Zakas
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Helper Functions
+//------------------------------------------------------------------------------
+
+function getMessageType(message) {
```

Comment:
Can you please add JSDoc here?  See https://github.com/eslint/eslint/blob/master/lib/formatters/tap.js#L13-L17 for an example.